autoscale/src/main/java/com/sequenceiq/periscope/monitor/evaluator/load/YarnResponseUtils.java

```diff
@@ -17,25 +18,34 @@ public class YarnResponseUtils {
 
```

Comment:
Overall, this class is performing quite a bit beyond processing Yarn Responses. For instance, any adjustments related to maxStepSize, or scale up as a result of maxNodeCount config - don't belong here.

Reply:
Removed unnecessary mandatory adjustment calculations from this class